spec/td/client/api_ssl_connection_spec.rb

```diff
@@ -47,6 +47,22 @@ def to_text
     }.to raise_error OpenSSL::SSL::SSLError
   end
 
+  it 'success to access to the server with verify false option' do
+    @server = setup_server(:TLSv1_2)
+    api = API.new(nil, :endpoint => "https://localhost:#{@serverport}", :retry_post_requests => false, :verify => false)
+    expect {
+        api.delete_database('no_such_database') # When openssl does not support SSLv3, httpclient server will not start. For context: https://github.com/nahi/httpclient/pull/424#issuecomment-731714786
+    }.to raise_error TreasureData::NotFoundError
+  end
+
+  it 'success to access to the server with self sighed certificate' do
+    @server = setup_server(:TLSv1_2)
+    api = API.new(nil, :endpoint => "https://localhost:#{@serverport}", :retry_post_requests => false, :verify => File.join(DIR, 'testRootCA.crt'))
+    expect {
+        api.delete_database('no_such_database') # When openssl does not support SSLv3, httpclient server will not start. For context: https://github.com/nahi/httpclient/pull/424#issuecomment-731714786
```

Comment:
Remove comment # When openssl does not support SSLv3, httpclient server will not start. For context: https://github.com/nahi/httpclient/pull/424#issuecomment-731714786